src/frontend-samples/hypermodeling-sample/HyperModelingApi.ts

```diff
@@ -41,11 +42,19 @@ class MarkerHandler extends SectionMarkerHandler {
     // One of the section drawings in the source model is bad - hide it.
     return marker.state.id !== "0x170c" && super.isMarkerVisible(marker, decorator, config);
   }
+
+  public async executeCommand(commandId: string, marker: SectionMarker, decorator: HyperModelingDecorator) {
+    if (commandId !== "apply_view") {
```

Comment:
Add comment for target audience indicating why you're doing this?

Reply:
Done.